DigitalExcellence/dex-frontend
# Feature/463 highlight images

## Description

Implement highlight images so they can be used in the highlight slider on the hompage.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Login as bob
2. Create a highlight for 3 projects and add images
3. Go to the hompage and check if the header is showing the images
4. Hovering over a card should activate that highlight
5. Edit one of the highlights and change the image
6. Check that the images changed in the header

## Link to issue

Closes: #463
